src/flypad-backend/src/ThirdParty/ThirdParty.h

```diff
@@ -0,0 +1,65 @@
+#pragma once
+
+#include <MSFS/Legacy/gauges.h>
+#include <MSFS/MSFS.h>
+#include <MSFS/MSFS_Render.h>
+#include <SimConnect.h>
+
+#include "FlyPadBackend.h"
+
+struct ThirdPartyDataIVAO {
+  uint8_t selcal;
+  uint8_t volumeCOM1;
+  uint8_t volumeCOM2;
+};
+
+struct ThirdPartyDataVPILOT {
+  uint8_t loaded; // Set to 1 if the aircraft is loaded. 0 once unloaded. If loaded, vPilot does not play the SELCAL sound
+  uint8_t selcal;
+};
+
+class ThirdParty{
```

Comment:
Why did you call it third party - and not Selcal? Is it generic enough to do other things? Do you plan to extend this?

If you think of a more general use then I think the SELCAL use case should still be commented somewhere so that a reader understand what the use case is for. All other modules' (LightPreset, AircractPreset, Pushback) use cases are clear from the name.

For me this is important - either clear names (classes, methods, vars, etc. ) bound to the use case or good comments. This way another devs reading this have a much easier time understanding it. I always do it as I forget everything after a few months anyway. 😆

Reply:
Renamed into ATCServices